### DISCUSSION TITLE
What's the naming convention for the struct that holds extend_ref of contract signer?

### DISCUSSION THREAD

**Discussion by 0x-j** [aptos-labs/aptos-developer-discussions#69] [Questions] [answered]
Take Aptogtochi as an example, we create an object and use it as the creator of the aptogotchi collection and token and store the extend_ref of the object in a [struct](https://github.com/aptos-labs/aptogotchi-intermediate/blob/j/update/move/sources/aptogotchi.move#L52-L60). I've seen sometimes people call the struct `Config`,  `MyRefs` or `AppCapability`, what;s the most unconfusing naming you guys think?

**Comment by gregnazario** [comment, marked as answer]
I wouldn't say there is a convention, since it often can be part of your application data directly.

I've been working with `ObjectController` or `ObjectRefs`

or say you were making an `Escrow` so you might call it `EscrowRefs`.
